app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi.json

```diff
@@ -1139,102 +1154,151 @@
+                                            "/widgets": {
+                                                "get": {
+                                                    "responses": {
+                                                        "200": {
+                                                            "content": {
+                                                                "application/json": {
+                                                                    "schema": {
+                                                                        "type": "array",
+                                                                        "items": {
+                                                                            "type": "string"
+                                                                        }
+                                                                    }
+                                                                }
+                                                            },
+                                                            "description": "All widgets"
+                                                        }
+                                                    },
+                                                    "summary": "Get widgets"
+                                                }
+                                            }
+                                        },
+                                        "components": {
+                                            "schemas": {
+                                                "Widget": {
+                                                    "title": "Root Type for Widget",
+                                                    "description": "A sample data type.",
+                                                    "type": "object",
+                                                    "properties": {
+                                                        "property-1": {
+                                                            "type": "string"
+                                                        },
+                                                        "property-2": {
+                                                            "type": "boolean"
+                                                        }
+                                                    },
+                                                    "example": {
+                                                        "property-1": "value1",
+                                                        "property-2": true
+                                                    }
+                                                }
+                                            }
+                                        }
+                                    }
+                                }
+                            }
+                        }
```

Comment:
Here's another request body that does not define a schema.  This is also the request body I mentioned in #1476

Reply:
Confirmed this is still causing `openapi-generator` to throw an exception:


```
Exception in thread "main" java.lang.RuntimeException: Could not process operation:
  Tag: class Tag {
    name: Metadata
    description: Sometimes the metadata for an artifact is important. For example, metadata includes 
    when the artifact was created, last updated, and so on. This section contains 
    operations to access (and in some cases change) an artifact's metadata.
    externalDocs: null
}
  Operation: getArtifactVersionMetaDataByContent
  Resource: post /groups/{groupId}/artifacts/{artifactId}/meta
  Schemas: {EditableMetaData=class ObjectSchema {
```